lupa/_lupa.pyx

```diff
@@ -1212,11 +1218,11 @@ cdef int py_function_result_to_lua(LuaRuntime runtime, lua_State *L, object o) e
      return py_to_lua(runtime, L, o)
 
 cdef int py_to_lua_handle_overflow(LuaRuntime runtime, lua_State *L, object o) except -1:
-    cdef const char* cb_name = "lupa_overflow_cb"
-    cdef int nargs = 0
+    cdef int nargs
 
-    lua.lua_getglobal(L, cb_name)
-    if lua.lua_isfunction(L, -1):
+    lua.lua_pushlstring(L, "LUPAOFH", 7)
```

Comment:
Unless we can avoid adding this global name entirely, let's at least make it long and explicit, to reduce the chances of a naming collision in Lua code.
For example: `_LUPA_NUMBER_OVERFLOW_CALLBACK_FUNCTION`. And please put it into a global constant, like `POBJECT` further up.

Reply:
It's actually in the register, not in the global scope!

Reply:
Resolved in 6fb99e6